doc-pages/src/components/examples.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import ExampleApp from '../../../examples/main/app';
+
+// TODO(abm): This is made into a wrapper to improve later
+
+export default class Example extends Component {
```

Comment:
Not sure I understand what this does.. Are we not going to have a list of separate clickable examples as the other repos do?